extensions/applicationinsights-angular-js/test-app/src/app/app.component.spec.ts

```diff
@@ -49,20 +52,6 @@ describe('Router: App', () => {
     });
   }));
 
-  it('Angular Configuration: Config options can be passed from root config', fakeAsync(() => {
-    init();
-    angularPlugin.initialize({
-      instrumentationKey: 'instrumentation_key',
-      extensionConfig: {
-        [angularPlugin.identifier]: {
-          router
-        }
-      }
-    }, core, []);
-    const angularConfig: IAngularExtensionConfig = angularPlugin['_extConfig'];
```

Comment:
Can we please also remove _extConfig from being exposed in the AngularPlugin as part of this PR